uu2nano.py

```diff
@@ -0,0 +1,40 @@
+import uuid
+
+
+__version__ = '1.0'
+
+
+def _make_alpharev(alphabet):
+    alpharev = bytearray(max(c for c in alphabet) + 1)
+    for i, c in enumerate(alphabet):
+        alpharev[c] = i
+    return bytes(alpharev)
+
+
+alphabet = b'_-0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ'
+alpharev = _make_alpharev(alphabet)
+_low_mask = 2 ** 62 - 1
+_const_bits = 0b10 << 62
+
+
+def uuid_to_nanoid(uu: uuid.UUID, *, alphabet=alphabet) -> str:
+    uu = uu.int
+    assert uu >> 62 & 0b11 == 2, "Wrong mark bits. Use fix_uuid() for true random input"
```

Comment:
I would not advertise `fix_uuid` here. It's a hack for specific case, not general way to make nanoid work for arbitrary input.